core/src/main/java/org/lflang/LinguaFranca.xtext

```diff
@@ -224,7 +224,7 @@ Watchdog:
     code=Code;
 
 STP:
-    'STP' '(' value=Expression ')' code=Code;
+    ('STP' | 'STAA') '(' value=Expression ')' code=Code;
```

Comment:
Suggestion for improvement: a deprecation notice for `STP` in the validator would make sense.

Reply:
I suggest we deprecate both syntaxes together when we come up with a better syntax and for now just allow either STP or STAA.  Otherwise, we need to change all the tests, playground examples, and documentation that use STP and STP_offset now, only to change them all again later.